examples/aggregate.php

```diff
@@ -0,0 +1,59 @@
+<?php
+declare(strict_types=1);
+
+namespace MongoDB\Examples;
+
+use MongoDB\Client;
+
+use function assert;
+use function dirname;
+use function getenv;
+use function is_object;
+use function MongoDB\BSON\fromPHP;
+use function MongoDB\BSON\toRelaxedExtendedJSON;
+use function printf;
+use function rand;
+
+require dirname(__FILE__) . '/../vendor/autoload.php';
```

Comment:
`__DIR__` should be a suitable replacement for `dirname(__FILE__)` and spare us the additional `use function` import.

See: https://www.php.net/manual/en/language.constants.magic.php`

Reply:
Done.